apps/selvbetjening-ui/src/inntektsjustering/3-oppsummering/InntektsjusteringOppsummering.tsx

```diff
@@ -7,23 +7,93 @@ import { InntektsjusteringOppsummering as InntektsjusteringOppsummeringInnhold }
 import { Navigate, useNavigate } from 'react-router-dom'
 import { NavigasjonMeny } from '../../common/NavigasjonMeny/NavigasjonMeny.tsx'
 import { useInntekt } from '../../common/inntekt/InntektContext.tsx'
+import { SkalGaaAvMedAlderspensjon } from '../../types/inntektsjustering.ts'
+import { useInnloggetInnbygger } from '../../common/innloggetInnbygger/InnloggetInnbyggerContext.tsx'
 import { finnAlder } from '../2-inntekt-til-neste-aar/finnAlder.ts'
 import { Alder } from '../../types/person.ts'
+import { format, Locale } from 'date-fns'
+import { Spraak } from '../../common/spraak/spraak.ts'
+import { nb, nn, enGB } from 'date-fns/locale'
+
+const spraakTilDateFnsLocale = (spraak: Spraak): Locale => {
+    switch (spraak) {
+        case Spraak.BOKMAAL:
+            return nb
+        case Spraak.NYNORSK:
+            return nn
+        case Spraak.ENGELSK:
+            return enGB
+        default:
+            return nb
+    }
+}
 
 export const InntektsjusteringOppsummering = () => {
-    const spraak = useSpraak()
     const navigate = useNavigate()
+
+    const spraak = useSpraak()
     const inntekt = useInntekt()
 
-    const { innhold, error, isLoading } = useSanityInnhold<InntektsjusteringOppsummeringInnhold>(
-        '*[_type == "inntektsjusteringOppsummering"]'
-    )
+    const {
+        data: innloggetBruker,
+        error: innloggetBrukerError,
+        isLoading: innloggetBrukerIsLoading,
+    } = useInnloggetInnbygger()
+
+    const {
+        innhold,
+        error: innholdError,
+        isLoading: innholdIsLoading,
+    } = useSanityInnhold<InntektsjusteringOppsummeringInnhold>('*[_type == "inntektsjusteringOppsummering"]')
+
+    if (innloggetBrukerError && !innloggetBrukerIsLoading && !innloggetBruker) {
+        return <Navigate to="/system-utilgjengelig" />
+    }
 
-    if (error && !isLoading) {
+    if (innholdIsLoading) {
+        return <Loader />
+    }
+    if (innholdError) {
         return <Navigate to="/system-utilgjengelig" />
     }
+    if (!innhold?.skjemaSammendrag) {
+        return <Navigate to="/system-utilgjengelig" />
+    }
+
+    // Hvis en person er over 56 og ikke har gitt stilling om hen skal gå av med alderspensjon, send tilbake til skjema for inntekt
+    if (finnAlder(innloggetBruker!) !== Alder.ATTEN_TIL_FEMTI_SEKS) {
+        if (inntekt.skalGaaAvMedAlderspensjon === undefined) {
+            return <Navigate to="/inntektsjustering/inntekt-til-neste-aar" />
```

Comment:
```suggestion
            return <Navigate to="/inntektsjustering/inntekt-til-neste-aar" replace />
```

er sikkert greit å overskrive history-entry'et, slik at back ikke tar deg i en redirect-loop